src/test/init/features/firestore.spec.ts

```diff
@@ -33,12 +41,25 @@ describe("firestore", () => {
     });
 
     it("should error when cloud resource location is not set", async () => {
+      checkApiStub.returns(true);
+
       const setup = { config: {}, projectId: "my-project-123" };
 
       expect(firestore.doSetup(setup, {})).to.eventually.be.rejectedWith(
         FirebaseError,
         "Cloud resource location is not set"
       );
     });
+
+    it("should error when the firestore API is not enabled", async () => {
+      checkApiStub.returns(false);
+
+      const setup = { config: {}, projectId: "my-project-123" };
+
+      expect(firestore.doSetup(setup, {})).to.eventually.be.rejectedWith(
+        FirebaseError,
+        "It looks like you haven't used Cloud Firestore"
```

Comment:
Did you want to finish this messaging?  or am I missing something?

Reply:
This test just looks for a prefix.